docs/operate/mobility/define-obstacles.md

```diff
@@ -5,5 +5,5 @@ weight: 20
 layout: "docs"
 type: "docs"
 no_list: true
-description: "TODO"
+description: "Spatially describe your robot's working environment for collision avoidance."
```

Comment:
robot or machine?

Reply:
Same as below; I think "robot" evokes the right image in this context.